### ISSUE TITLE
Loot is not triggered on killing Pristess Delrissa on Magister Terrace (BB #1023)

### ISSUE THREAD

**Issue by OregonAsari** [OregonCore/OregonCore#1023]
This issue was migrated from bitbucket.
**Original Reporter:** silvermcd123
**Original Date:** 23.06.2015 18:02:15 GMT+0000
**Original Priority:** major
**Original Type:** bug
**Original State:** invalid
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/1023

<hr>

![WoWScrnShot_062315_190030.jpg](https://bitbucket.org/repo/kMeMB/images/483316585-WoWScrnShot_062315_190030.jpg)

id: 24560

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** desteny
**Original Date:** 23.06.2015 21:35:25 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/1023/#comment-19242136

<hr>

you used .die command?

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** LordUsagi
**Original Date:** 24.06.2015 01:23:22 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/1023/#comment-19244627

<hr>

We require all bug reports to be tested by NON-GM characters.

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** PadreWoW
**Original Date:** 24.06.2015 06:02:53 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/1023/#comment-19247523

<hr>

There should be no loot until you kill all its guards

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** silvermcd123
**Original Date:** 24.06.2015 15:32:15 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/1023/#comment-19263911

<hr>

@LordUsagi i know that and was tested by non-gm characters

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** silvermcd123
**Original Date:** 24.06.2015 15:32:52 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/1023/#comment-19263928

<hr>

@desteny i used .damage

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** LordUsagi
**Original Date:** 24.06.2015 15:36:28 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/1023/#comment-19264020

<hr>

Just tested on normal mode, after killing lackeys loot works. About to try heroic.
Confirmed on heroic too, lackeys must be destroyed.

```
#!c++

    void JustDied(Unit* /*killer*/)
    {
        DoScriptText(SAY_DEATH, me);

        if (!instance)
            return;

        if (instance->GetData(DATA_DELRISSA_DEATH_COUNT) == MAX_ACTIVE_LACKEY)
            instance->SetData(DATA_DELRISSA_EVENT, DONE);
        else
        {
            if (me->HasFlag(UNIT_DYNAMIC_FLAGS, UNIT_DYNFLAG_LOOTABLE))
                me->RemoveFlag(UNIT_DYNAMIC_FLAGS, UNIT_DYNFLAG_LOOTABLE);
        }
    }
```

You'll want to alter this to allow for killing her directly, I'll do some research once the conditions branch is merged into the repository and amend it if it requires.

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** LordUsagi
**Original Date:** 24.06.2015 15:39:30 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/1023/#comment-19264108

<hr>

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** PadreWoW
**Original Date:** 25.06.2015 06:02:26 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/1023/#comment-19276119

<hr>

I said it 23 hours ago.....

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** LordUsagi
**Original Date:** 28.06.2015 21:14:53 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/1023/#comment-19350151

<hr>

Padrewow was correct.
